Source/OnlineSubsystemEpic/Private/OnlinePrensenceEpic.cpp

```diff
@@ -478,8 +481,7 @@ EOnlineCachedResult::Type FOnlinePresenceEpic::GetCachedPresence(const FUniqueNe
 				// Get the last time the querying user was online.
 				FString lastOnlineString;
 				userAcc->GetUserAttribute(USER_ATTR_LAST_LOGIN_TIME, lastOnlineString);
-				OutPresence->LastOnline = FDateTime::FromUnixTimestamp(FCString::Atoi64(*lastOnlineString));
-
+				
```

Comment:
Why was the last online time removed? It's never added anywhere else.

Reply:
LastOnline is not a field that exists in the FOnlinePresence struct. I'll put it back in but comment it out and explain what I just said there.

Reply:
Actually it is. In OnlinePresenceInterface.h L184 `LastOnline` is declared.

Reply:
Ah, I see where the confusion is. It is _not_ a field in 4.24.3, but is in 4.25. It doesn't seem to be used right now though, can that just be removed for now though? The only place it seems to be used is in SocialUser in the Party class and that doesn't seem that it will be implemented for a while.